src/libs/ReportUtils.js

```diff
@@ -481,6 +483,24 @@ function getReportName(report, personalDetailsForParticipants = {}, policies = {
     return _.map(displayNamesWithTooltips, ({displayName}) => displayName).join(', ');
 }
 
+/**
+ * Navigate to the details page of a given report
+ *
+ * @param {Object} report
+ * @param {Array} participants
+ */
+function navigateToDetailsPage(report, participants) {
```

Comment:
Do we need to pass `participants` can we get it from `report` itself?

Clean this up, please!